### ISSUE TITLE
Is otter vulnerable to spectre?

### ISSUE THREAD

**Issue by AcarBurak** [OtterBrowser/otter-browser#1482]
If it is, is there any mitigation plan?

**Comment by Frenzie** [comment]
Isn't that more of a thing for Windows/Linux to deal with or am I confused with Meltdown?

**Comment by AcarBurak** [comment]
Almost anything having intel (Linux, Mac, Windows, BSDs etc.) is vulnerable to Meltdown. And almost anything + browsers are vulnerable to Spectre. Both [Firefox](https://www.ghacks.net/2017/11/22/how-to-enable-first-party-isolation-in-firefox/) and [Chrome](https://support.google.com/chrome/answer/7623121?hl=en-GB) has some mitigation plans some of which have already been implemented.

**Comment by pierreporte** [comment]
@Frenzie Yes and no. I’ve read that wether the OS is patched or not, a simple JS code would be able to read inputs and outputs of every single opened tab.

**Comment by beastie1** [comment]
They have mitigation plans probably because they are also the main developers of their respective engines. We'll have to wait for a patch for revived WebKit...?!?!

**Comment by AcarBurak** [comment]
For qtwebengine:

https://bugreports.qt.io/browse/QTBUG-65561

> Looking at the recent Meltdown/Spectre attacks, it'd also be good to support Site Isolation (site-per-process, not to be confused with process-per-site) as it's the recommended mitigation at the moment. Currently it seems to be possible to pass that switch to QtWebEngine, but I'm not sure whether it actually works correctly.

**Comment by AcarBurak** [comment]
https://webkit.org/blog/8048/what-spectre-and-meltdown-mean-for-webkit/

**Comment by paradogz** [comment]
Commit in qtwebkit which reduced timer precision: http://code.qt.io/cgit/qt/qtwebkit.git/commit/?h=5.6&id=d281c107389f43edb8ecdf47c4f49bd330837232

**Comment by AcarBurak** [comment]
Thank you.
